src/stylesheets/components/_footer.scss

```diff
@@ -341,7 +341,7 @@ li.usa-footer-primary-content {
   }
 
   ul {
-    padding-bottom: 2.5rem;
+    padding-bottom: rem(24px);
```

Comment:
This `rem(24px)` mixin technique was removed from use in https://github.com/18F/web-design-standards/pull/2006 due to Bourbon deprecation and loss of support in Bourbon 5.0, so I think we shouldn't use this.

Reply:
I remember when this happened, and I'm comfortable removing this function, _but_ I still think the Bourbon deprecation is a mistake and writing in raw `rem` makes the system too dependent on its base size declaration (10px). That is, if the System has to play nice with an existing codebase that's already defined its base size, we're SOL. Adding our own function to replace the soon-to-be-deprecated Bourbon function would make the whole system more flexible. But perhaps this is an argument for another day!